ansibledoctor/cli.py

```diff
@@ -68,6 +68,14 @@ def _cli_args(self):
         parser.add_argument(
             "--version", action="version", version="%(prog)s {}".format(__version__)
         )
+        parser.add_argument(
+            "-s",
+            "--skip-role-detection",
```

Comment:
According to `-n` I  would like to rename the flag/variables to `--no-role-detection`